e2e-tests/resources/demo.txt

```diff
@@ -7,7 +7,7 @@ On September 18 I visited microsoft.com and sent an email to test@presidio.site,
 
 My passport: 191280345 and my phone number: (212) 555-1234.
 
-This is a valid IBAN: IL150120690000003111111 . Can you please check the status on bank account 954567876544?
+This is a valid International Bank Account Number: IL150120690000003111111 . Can you please check the status on bank account 954567876544?
```

Comment:
Needs to be changed in the website as well

Reply:
Worth doing a PR and linking it here so we won't forget.